Dockerfile

```diff
@@ -1,23 +1,11 @@
 FROM node:lts as build-stage
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-# Install deps
-RUN apt-get update -y \
-    && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get install --no-install-recommends -y wget=1.* \
-    && rm -rf /var/lib/apt/lists/*
-# Download latest nvim stable Github release. See:
-# https://gist.github.com/steinwaywhw/a4cd19cda655b8249d908261a62687f8
-RUN wget -O - --progress=dot:giga https://api.github.com/repos/neovim/neovim/releases/latest \
-    | grep 'browser_download_url.*nvim.appimage"' \
-    | cut -d : -f 2,3 \
-    | tr -d \" \
-    | wget --progress=dot:giga -i -
-RUN chmod +x nvim.appimage
-RUN ./nvim.appimage --appimage-extract
-RUN chmod -R a+rx /squashfs-root/
-RUN mkdir -p /usr/local/bin
-RUN ln -s /squashfs-root/AppRun /usr/local/bin/nvim
+# Install Neovim
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update -y && \
+    apt-get install --no-install-recommends -y neovim && \
```

Comment:
I couldn't find this information, but depending on what debian version the docker image is using, this might install a [neovim version < 0.4](https://packages.debian.org/search?keywords=neovim&searchon=names&suite=all&section=all). I would prefer to avoid this, even if it should be fine in practice as the docker image's neovim is only used to install the manifest.

Can you confirm that the node:lts image is using debian bullseye or more recent?

Reply:
Um, no, it appears to be Debian *stretch* which has NeoVIM 0.1.7. Standby for something fresher…

Reply:
> Hey @infokiller was there a particular need to use an unstable/nightly NeoVIM build? If so there are normal non-Appimage packages for that too, but I didn't see a need to break out of the stable repository packages here.

This is not a nightly/unstable build, it's the stable one, just try it in a terminal.

Reply:
I wouldn't use the PPA if it's not maintained really well. PPAs are a security risk (probably more than Github releases in this case?). Also, the PPA only supports Debian-based distros, and not old versions that nodejs images may be using, so it may become an issue.

Reply:
The latest Debian base available in the node container family is Buster which has 0.3.x. I found a way to download and install newer packages from non-distro sources, but I don't see the point. Node also has Alpine Linux based images which are smaller, have package management features much more suited to Docker, and the NeoVIM version is 0.4.4.

I used the `node:lts-alpine` image, but there is also a `node:current-alpine` if you'd rather it build with a newer Node version.

Reply:
LGTM on using Alpine